client/app/services/vms.service.js

```diff
@@ -46,6 +46,8 @@ export function VmsService(CollectionsApi, RBAC) {
         'compliances',
         'cpu_affinity',
         'created_on',
+        'custom_actions',
```

Comment:
I belieeeeeeve adding these new attributes will break mock api get requests... probably should update that 😬